test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,22 @@
-const { fizzBuzz } = require("../src");
+const {fizzBuzz} = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toBe('Fizz');
```

Comment:
Could a few more valid assertions be included here to make the test more robust? Same goes for the following test too